pom.xml

```diff
@@ -516,13 +517,13 @@
             <dependency>
                 <groupId>com.h2database</groupId>
                 <artifactId>h2</artifactId>
-                <version>2.2.220</version>
+                <version>2.2.224</version>
             </dependency>
 
             <dependency>
                 <groupId>com.microsoft.sqlserver</groupId>
                 <artifactId>mssql-jdbc</artifactId>
-                <version>12.2.0.jre11</version>
+                <version>12.4.1.jre11</version>
```

Comment:
We should switch to the jre 17 version...

Reply:
There is no such build :)

Reply:
oh wow .. when did they switch that off and remove it again. I missed that.

Reply:
Turns out they dropped it in the 12.x series .. https://search.maven.org/artifact/com.microsoft.sqlserver/mssql-jdbc